tests/test_risk_distributions.py

```diff
@@ -6,10 +6,11 @@
 
 distributions = [risk_distributions.Exponential, risk_distributions.Gamma, risk_distributions.Gumbel,
                  risk_distributions.InverseGamma, risk_distributions.InverseWeibull, risk_distributions.LogLogistic,
-                 risk_distributions.LogNormal, risk_distributions.Normal, risk_distributions.Weibull]
+                 risk_distributions.LogNormal, risk_distributions.Normal, risk_distributions.Weibull,
+                 risk_distributions.Beta]
```

Comment:
oh cool, so there was a test that would have caught this?

Reply:
I hope the Mirrored Gumbel and Gamma are not making trouble.  At least we know their means are right...

Reply:
> oh cool, so there was a test that would have caught this?
Yes, it was masked by a problem with the process function. I include the Beta dist and the test failed. I then added your changes and the test passed.